routes/web.php

```diff
@@ -16,3 +16,9 @@
 Route::get('/', function () {
     return view('welcome');
 });
+Route::get('/about', function () {
+    return view('about');
+});
+Route::get('/contacts', function () {
```

Comment:
Replace the closure with a proper PageController

Reply:
Ok, working on it!